cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -288,14 +300,19 @@ func generateMountsModeRun() error {
 			// check for the try kernel
 			tryKernel, tryKernelExists, err := ebl.TryKernel()
 			// TODO:UC20: can we log somewhere if err != nil here?
-			if tryKernelExists && err == nil {
-				kernel = tryKernel
+			// TODO:UC20: can we log somewhere if this kernel snap isn't in the
```

Comment:
Hmm, I don't think we want to do that, since the output from `snap-bootstrap initramfs` is fed into a file that `the-tool` executes, so there's a possibility that this will break. See the definition of `the-tool`:

```bash
# Run snap-bootstrap
cmd="systemd-mount --no-pager --no-ask-password "
/usr/lib/snapd/snap-bootstrap initramfs-mounts | sed "s/^/$cmd/" > /root/bootstrap.cmds.$1
set +e
. /root/bootstrap.cmds.$1
set -e
```

(from https://git.launchpad.net/ubuntu-core-initramfs/tree/factory/usr/lib/the-tool#n4)